turbo.json

```diff
@@ -1,84 +1,87 @@
 {
```

Comment:
Just a note that prettier formatted this file. Afaict we are still using prettier in this repo so this shouldn't change again? Unless I have my prettier somehow misconfigured.